packages/app/src/app/hooks/useInterval.ts

```diff
@@ -0,0 +1,21 @@
+// Based on https://overreacted.io/making-setinterval-declarative-with-react-hooks/
+import { useEffect, useRef } from 'react';
+
+export const useInterval = (callback = () => {}, delay: number) => {
+  const savedCallback: { current: any } = useRef();
```

Comment:
No need to type `savedCallback` here.

Reply:
Not destructuring here, though we possibly could. Just adding a type to satisfy the linter. Probably clean this up later as we make another pass to flesh out types across the code base.